tests/spec/core/data-include-spec.js

```diff
@@ -81,4 +88,38 @@ describe("Core — Data Include", () => {
     expect(h2.textContent).toBe("1. PASS");
     expect(doc.querySelectorAll("*[data-include]").length).toBe(0);
   });
+
+  it("indents multiline text", async () => {
+    const include = `Blue rose is blue
+
+Red rose is red
+
+Rose of the wasteland is violet
+`;
+    const ops = {
+      config: {
+        ...makeBasicConfig(),
+        format: "markdown",
+      },
+      body: `
+        ## R
+
+        1. Rose
+
+          <div data-include="${generateDataUrl(
```

Comment:
This is.... insane... I hope no one actually does this in practice! But very cool that it works.

Reply:
Insane, yes 😂

Although this makes more sense in more real-world like example:

```
  <section id="sotd">...</section>

  <section data-include="..."></section>
```

Previously no indentation was applied and thus the replacement text went indentation-less, causing incorrect markdown. I couldn't add this test here though, as `normalizePadding` somehow silences it.

Reply:
Make sense.